### ISSUE TITLE
Unreal Editor spammed with error 16

### ISSUE THREAD

**Issue by bbuckley94** [disguise-one/RenderStream-UE#43]
**Summary:**
When the RenderStream plugin is added to a project, the output log is spammed with the following error when the Editor is launched:

```
[2021.12.22-09.07.07:879][115]LogRenderStream: Error: Error awaiting frame data error 16
[2021.12.22-09.07.07:895][116]LogRenderStream: Error: Workload asset not created
RenderStream Callstack:  < rs_awaitFrameData < RSAwaitFrameData
```

With the current retail version of d3 and UE plugin, the callstack is blank. When testing with the tip of r20.3 and r1.28_UE4.27 the callstack information is included.

**Bug reduction:**
- Seen in r20.0.3 (84705) with RS1.28 (UE 4.27) v2 - latest public release.
- Seen in r20.3 (84847) with RenderStream-UE_r1.28_UE4.27_r1.28_UE4.27_r20.0_v2-3-g4eb863a (Patches Applied).

**Steps to reproduce:**
1. Launch UE and create a new blank project
2. Add the plugin to the project folder as normal
3. Restart UE
4. Open the output log (notice error spam)

**Comment by MunWolf** [comment]
Fixed in 5176d28d994d92073d0f8723de2d237e3d454a42

**Comment by MunWolf** [comment]
This is assuming we are only encountering the issue when running outside of a workload, if it also happens in a workload please make a new issue as it will not be happening for the same reason.
